ztp/gitops-subscriptions/argocd/README.md

```diff
@@ -325,3 +325,19 @@ Check labels on MangedCluster:
     $ oc get policy -n $CLUSTER
 ```
 If the Namespace, OperatorGroup, and Subscription policies are compliant but the operator configuration policies are not it is likely that the operators did not install on the spoke cluster. This causes the operator config policies to fail to apply because the CRD is not yet applied to spoke.
+
+### Restart Policies reconciliation
+A ClusterGroupUpgrade CR is generated in the namespace ```ztp-install``` by Topology Aware Lifecycle Operator after the managed spoke cluster becomes ```Ready```:
+```
+     $ export CLUSTER=<clusterName>
+     $ oc get clustergroupupgrades -n ztp-install $CLUSTER
+```
+If unexpected issues happen that cause the Policies fail to become complaint within the configured timeout(default is 4h), the status of ClusterGroupUpgrade CR will show ```UpgradeTimedOut```:
+```
+     $ oc get clustergroupupgrades -n ztp-install $CLUSTER -o jsonpath='{.status.conditions[?(@.type=="Ready")]}'
+```
+A retry of policies reconciliation will be triggered after 1h when ```UpgradeTimedOut``` happens. Once you have investigated/fixed the issues, you can trigger the reconciliation immediately by deleting the existing ClusterGroupUpgrade CR, this will make a new ClusterGroupUpgrade CR to be created automatically:
```

Comment:
Suggestion to rephrase at the end:
". This will automatically trigger the creation of a new ClusterGroupUpgrade CR."